### DISCUSSION TITLE
希望官网补充vue3+vite使用文档

### DISCUSSION THREAD

**Discussion by liytingtt** [umijs/qiankun#2903] [General]
# 提问之前强烈建立您能先阅读一下[《如何正确的提出一个 Issue》](https://github.com/umijs/qiankun/issues/1115)

## 最小可复现仓库

为节约大家的时间，无复现步骤的 ISSUE 会被关闭，提供之后再 REOPEN

## 相关环境信息

- **qiankun 版本**
- **浏览器版本**：
- **操作系统**：

**Comment by tianhailiang** [comment]
官方大哥们 更新一个vue3 + vite 文档的例子吧

**Comment by QixuanHuang** [comment]
+1
目前项目使用vue3+ts+vite，找不到靠谱的教程
